src/views/Delegate.vue

```diff
@@ -157,7 +157,7 @@ async function getDelegatesWithScore() {
   delegatesLoading.value = true;
   try {
     const delegationsRes = await Promise.all([
-      getDelegatesBySpace(space.value.network, ''),
+      getDelegatesBySpace(space.value.network, null),
       getDelegatesBySpace(space.value.network, space.value.id)
     ]);
```

Comment:
Not sure I understand this. Isn't the space param unnecessary? It calls it with and without a space and then merges the results together. But the function is called `getDelegatesBySpace`, so I'm a bit confused.

Reply:
There are global delegations and space level delegations. To get both we need to query like this.

Reply:
Space global delegations will not be included if queried by space and vise versa.

Reply:
Yea I mean.... can't you just remove the where? As far as I can see this function is used only here for this one purpose.

Reply:
Oh I just saw that this is just meant to hotfix stable... approved. :D